nuxt.config.js

```diff
@@ -46,7 +46,7 @@ export default {
   /*
    ** Global CSS
    */
-  css: [],
+  css: ["@mdi/font/css/materialdesignicons.css"],
```

Comment:
please remove this. The way icons are included is with mdijs.  You can see how other buttons are in the template; To see the icons you can install this extension https://chrome.google.com/webstore/detail/materialdesignicons-picke/edjaedpifkihpjkcgknfokmibkoafhme. or directly from the website https://materialdesignicons.com/ (much slower). The reason is that the mdijs tree shake in compilation time, so you don't have to import all the icons (from the font)https://github.com/NLeSC/nuxt-apollo-hasura/blob/5eeb344209290daa43c42106dc5f4479e7141bdd/layouts/default.vue#L76